tests/integrations/config/notification.py

```diff
@@ -0,0 +1,21 @@
+"""Notifications Settings."""
+import os
+
+DRIVERS = {
+    "slack": {
+        "token": os.getenv("SLACK_TOKEN", ""),  # used for API mode
+        "webhook": os.getenv("SLACK_WEBHOOK", ""),  # used for webhook mode
+        # "mode": os.getenv("SLACK_MODE", "webhook"),  # webhook or api
+    },
+    "vonage": {
+        "key": os.getenv("VONAGE_KEY", ""),
+        "secret": os.getenv("VONAGE_SECRET", ""),
+        "sms_from": os.getenv("VONAGE_SMS_FROM", "+33000000000"),
+    },
+    "database": {
+        "connection": "sqlite",
+        "table": "notifications",
+    },
+}
+
+DRY = False
```

Comment:
Whats this thing?

Reply:
This is to globally disable notifications sending. I have found it useful when running commands in production for example. This is just a global switch controlled by a setting / environment variable